substrait_consumer/tests/functional/relations/project_relation_snapshots/DuckDBProducer/project_single_col_plan.json

```diff
@@ -0,0 +1,369 @@
+                      },
+                      {
+                        "date": {
+                          "nullability": "NULLABILITY_NULLABLE"
+                        }
+                      },
+                      {
+                        "date": {
+                          "nullability": "NULLABILITY_NULLABLE"
+                        }
+                      },
+                      {
+                        "date": {
+                          "nullability": "NULLABILITY_NULLABLE"
+                        }
+                      },
+                      {
+                        "string": {
+                          "nullability": "NULLABILITY_NULLABLE"
+                        }
+                      },
+                      {
+                        "string": {
+                          "nullability": "NULLABILITY_NULLABLE"
+                        }
+                      },
+                      {
+                        "string": {
+                          "nullability": "NULLABILITY_NULLABLE"
+                        }
+                      }
+                    ],
+                    "nullability": "NULLABILITY_REQUIRED"
+                  }
+                },
+                "projection": {
+                  "select": {
+                    "structItems": [
+                      {},
+                      {
+                        "field": 1
+                      },
+                      {
+                        "field": 2
+                      },
+                      {
+                        "field": 3
+                      },
+                      {
+                        "field": 4
+                      },
+                      {
+                        "field": 5
+                      },
+                      {
+                        "field": 6
+                      },
+                      {
+                        "field": 7
+                      },
+                      {
+                        "field": 8
+                      },
+                      {
+                        "field": 9
+                      },
+                      {
+                        "field": 10
+                      },
+                      {
+                        "field": 11
+                      },
+                      {
+                        "field": 12
+                      },
+                      {
+                        "field": 13
+                      },
+                      {
+                        "field": 14
+                      },
+                      {
+                        "field": 15
+                      }
+                    ]
+                  },
+                  "maintainSingularStruct": true
+                },
+                "namedTable": {
+                  "names": [
+                    "lineitemsmall"
+                  ]
+                }
+              }
+            },
+            "expressions": [
```

Comment:
This should double all of the output fields which means the output names are going to be wrong.